content/about/governance/index.en.md

```diff
@@ -5,28 +5,28 @@ description: Governance and organization of OSRD
 weight: 40
 ---
 
+OSRD is currently developed by a [team of full-time developers at SNCF Réseau](../../join-us), financed by SNCF Réseau, the French state and the European Union.
+
 One of the ambitions of OSRD is to help with **timetabling on a European scale**.
 
-To reach this goal, OSRD has to take local features into account by consulting and collaborating with railway companies and infrastructure operators.
+To reach this goal, OSRD has to take local features into account by consulting and collaborating with railway companies, infrastructure operators, research teams and any other interested actors.
 
-And by contributing to OSRD, these companies:
+If you are a company/team wanting to:
 
 - Profit from all the common development efforts
-- Can freely build on the common effort to achieve their own goals[^license]
-- Participate in the roadmap and technical direction of the project to the extent of their contribution
-
-[^license]: As long as they don't modify the common tools without sharing their improvements. See the details of the [LGPLv3](https://www.gnu.org/licenses/lgpl-3.0.en.html) license.
-
-OSRD is currently developed by:
+- Freely build on the common effort to achieve your own goals[^license]
+- Participate in the roadmap and technical direction of the project to the extent of your contribution
 
-- A [team of full-time developers at SNCF Réseau](../../join-us), financed by SNCF Réseau, the French state and the European Union
-- Individual contributors
+[^license]: As long as you share the improvements of common tools, see the details of the [LGPLv3](https://www.gnu.org/licenses/lgpl-3.0.en.html) license.
+ 
+Please get in touch with us at <contact@osrd.fr> !
 
-## The OpenRail association
+## OpenRail Association
 
-The [OpenRail association](https://openrailassociation.org/) is an upcoming non-profit committed to organizing cooperative work around railway software tools.
+The [OpenRail Association](https://openrailassociation.org/) is a non-profit co-founded by SNCF Réseau committed to organizing cooperative work around open-source railway software tools.
+OSRD is the first project hosted by the OpenRail Association and we work alongside another OpenRail project: [NGE](../../docs/explanation/netzgrafik-editor/).
 
-<a href="https://openrailassociation.org/"><img class="marginauto" src="logo-openrail-association.png" alt="OpenRail association"></a>
+<a href="https://openrailassociation.org/"><img class="marginauto" src="logo-openrail-association.png" alt="OpenRail Association"></a>
```

Comment:
This does not render as an image

<img width="1511" height="179" alt="Image" src="https://github.com/user-attachments/assets/d73f3726-a7af-41ba-b5ce-7573f8e506c4" />